lib/Spec.AVLTree.fst

```diff
@@ -0,0 +1,592 @@
+module Spec.AVLTree
```

Comment:
For this file specifically, please add a comment at the top that this file is copied verbatim from the steel repository.

Reply:
comment addressed.